ocp/cosa-bc.yaml

```diff
@@ -54,6 +54,14 @@ objects:
   - kind: ServiceAccount
     name: ${SA}
 
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
```

Comment:
In the Gangplank-in-Prow use case, Prow will create a temporary CI namespace for us - would this `ImageStream` be scoped to that?

In the Gangplank-as-periodic-builder-for-production aka Gangplank-replacing-Jenkins case (hmm these cases need a short name, wait...wait...I just made up "Gangprow" for the first one and am proud of that!  For this one...hmmm maybe "Prodplank"?) would we have a namespace per release stream e.g. separate `art-rhcos-4.7` namespaces, or would we just push different tags to the same imagestream?

Reply:
Thus it is written -- "Prodplank" and "Gangprow"

For that Gangprow periodics, I'm working with the CI-tools team to introduce custom-build-strategies (https://github.com/openshift/ci-tools/pull/1898 which may be different)  so we can use ephemeral namespaces. We won't be able to fully use this until we get the secret handling and role assignments for an ephemeral service account. In fact, this PR is the direct result of their request that the custom build produces a container. 

For Prodplank, I honestly haven't gotten that far due to the Brew requirements (which have been reinforced by product security). I *think* that we might need to do behind the firewall builds using Gangplank via `oc start-build bc/....` commands in Jenkins. 

`ocp/cosa-bc.yaml` is useful for Gangplank development and developer-one-offs.